code/index.html

```diff
@@ -2,21 +2,87 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hooked News</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
     <header>
       <!-- Some kind of logo and navbar -->
+      <div></div>
+      <div></div>
+      <div></div>
+      <img src="https://thumbs.dreamstime.com/t/breaking-news-minimalistic-logo-white-background-entertaining-show-news-vector-illustration-breaking-news-headline-113066889.jpg" alt="news logo">
+      <nav>
+        <a href="#">Sports</a>
+        <a href="#">Politics</a>
+        <a href="#">USA</a>
+      </nav>
     </header>
 
     <section class="big-news">
-      <!-- The big news content -->
+      <section class="news">
+        <div> 
+          <img src="https://i0.wp.com/calmatters.org/wp-content/uploads/2019/09/1024px-Donald_Trump_8566730507_2.jpg?fit=1024%2C683&ssl=1" alt="pictuer of someone">
+         <h1>Breathing oxygen linked to staying alive!</h1>
+         <p>
+          Reprots is saying that the former president of the United States has finally found a soloution for the survivability of the human race.
+         </p>
+        </div>
+        
+      </section>
+       <!-- The big news content -->
+     
     </section>
 
-    <section class="">
+    <section class="articles">
+      <section class="artis">
+        <h2>Better News</h2>
+        <section class="art">
+        <a href="#">
+          <div>
+            <img  src="https://creamies.com/wp-content/uploads/2020/01/Overrun-why-ice-cream-melts-so-quickly-.jpg" alt="pictue of ice cream">
+            <p>Ice Cream In Northeastern China Tests Positive For COVID-19</p>
+          </div>
+         </a>
```

Comment:
In this `<a>` tag you don't need the `<div>`, it's enough to wrap the `<a>` around the image and the paragraph.